news-frontend/src/components/App/App.js

```diff
@@ -1,35 +1,34 @@
 import React from "react";
 import { Switch, Route, Redirect } from "react-router-dom";
 import "./App.css";
-import Header from "../Header/Header";
+//import Header from "../Header/Header";
 import Main from "../Main/Main";
 import Footer from "../Footer/Footer";
 import SavedNewsHeader from "../SavedNewsHeader/SavedNewsHeader";
 import LoginPopup from "../LoginPopup/LoginPopup";
 import RegisterPopup from "../RegisterPopup/RegisterPopup";
 import InfoToolTip from "../InfoToolTip/InfoToolTip";
 import MobileMenu from "../MobileMenu/MobileMenu";
+import ProtectedRoute from "../ProtectedRoute/ProtectedRoute";
 
 function App() {
   return (
     <div className="app">
       <Switch>
+      <ProtectedRoute path="/saved-news">
```

Comment:
You need to format the markup to show indentation and to make the code more readable. You can do that with `prettier` or a combination `SHIFT + ALT + F`


```jsx


  return (
    <div className="app">
      <Switch>
        <ProtectedRoute path="/saved-news">
          <SavedNewsHeader />
        </ProtectedRoute>
        <Route exact path="/">
```